src/GitHubCostVisualizer.Web/Models/UsageReportViewModel.cs

```diff
@@ -30,12 +30,21 @@ public class UsageReportViewModel
         public int TotalBillableActionMinutes { get; set; }
         public decimal TotalActionMinutesCost { get; set; }
         public List<KeyValuePair<string, int>> ActionsSummary { get; set; }
-        public List<KeyValuePair<string, int>> ActionMinutesByRepository { get; set; }
-        public List<KeyValuePair<string, int>> ActionMinutesByWorkflow { get; set; }
+        public List<ActionMinutesItem> ActionMinutesByRepository { get; set; }
+        public List<ActionMinutesItem> ActionMinutesByWorkflow { get; set; }
         public decimal AverageDailyStorage { get; set; }
         public List<KeyValuePair<DateTime, decimal>> DailyStorageSummary { get; set; }
         public List<KeyValuePair<string, decimal>> AverageDailyStorageByRepo { get; set; }
-
         public DailyStorageData DailyStorageByRepo { get; set; }
     }
+
+    public class ActionMinutesItem
```

Comment:
Given that you only have 1 string at any given time, lets go with `public string Label { get; set; }` rather than a separate property for Repository or Workflow for this object

Reply:
Done